ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -193,15 +193,11 @@ objects:
         trap 'rc=$?; if [[ $rc -ne 0 ]]; then touch /tmp/shared/exit; else touch /tmp/shared/install; fi; exit $rc' EXIT
         trap 'kill $(jobs -p); exit 0' TERM
 
-        # Don't expand IMAGE_FORMAT here - it contains '${component}'
-        oreg_url='${IMAGE_FORMAT}'
-        if [[ -n "${PREVIOUS_IMAGE_TAG}" ]]; then
-          oreg_url='docker.io/openshift/origin-${component}:${PREVIOUS_IMAGE_TAG}'
-        fi
-
         ansible-playbook -vv \
           -e openshift_test_repo="${PREVIOUS_RPM_REPO}" \
-          -e oreg_url="${oreg_url}" \
+          -e oreg_url='docker.io/openshift/origin-${component}:v${PREVIOUS_ANSIBLE_VERSION}' \
```

Comment:
This is a strange way of setting oreg_url and not typically how we do it.  It might break some origin components such as cockpit which use ':latest'

Probably not necessary to set oreg_url at all unless it's being set to some other value somewhere else and we're just overriding it here.

Reply:
Right, since `_version` is set oreg_url is not required. Removed